services/management/service_test.go

```diff
@@ -146,5 +147,115 @@ func TestServiceService(t *testing.T) {
 			assert.Nil(t, service)
 			tests.AssertGRPCError(t, status.New(codes.NotFound, `Service with ID "/service_id/00000000-0000-4000-8000-000000000005" not found.`), err)
 		})
+
+		t.Run("RDS", func(t *testing.T) {
```

Comment:
both 2 cases are almost identical, let's extract a function and use it for test cases.

Reply:
Well, I thought about that, but it looks that it's easier to read/write/understand it's when it's plain copy-paste.